### ISSUE TITLE
rollback crashes on new record.

### ISSUE THREAD

**Issue by DuckoDeath** [danielspaniel/ember-data-change-tracker#45]
rollback() fails on a new record.
The error is:
_You must include an 'id' for oil/admin/company in an object passed to 'push'_

due to:
``javascript
this.store.push(normalized);
``
at the end of the rollback function.

A weird use-case I know, have a new dialog where the user can revert back to the original state (without saving)

**Comment by danielspaniel** [comment]
hold the phone for second, i looking into it

**Comment by danielspaniel** [comment]
fixed in https://github.com/danielspaniel/ember-data-change-tracker/releases/tag/v0.7.6 
thanks for the issue 👍

**Comment by sylwiaeb** [comment]
@danielspaniel I just got the same error when rolling back on a new record. In my case, the reason for error was that I called `model.rollbackAtributes()` before calling `model.rollback()`. 
After removing `rollbackAtributes` it works fine though my question is: with this addon I shouldn't have to call `rollbackAtributes` right? `rollback` will take care of discarding attributes and relationships (if enabled).
